src/reducers/searchValues.js

```diff
@@ -0,0 +1,85 @@
+import searchActions from '../actions/searchValues'
```

Comment:
I think this reducer could be split in two. Even if all the actions are defined in `searchActions`, some of them are concerned with I/O and others with the search value.